ctapipe/reco/tests/test_FitGammaHillas.py

```diff
@@ -0,0 +1,62 @@
+from astropy import units as u
+
+from ctapipe.instrument.InstrumentDescription import load_hessio
+
+from ctapipe.utils.datasets import get_path
+
+from ctapipe.reco.FitGammaHillas import FitGammaHillas
+from ctapipe.image.hillas import hillas_parameters
+from ctapipe.image.cleaning import tailcuts_clean, dilate
+
+from ctapipe.io.hessio import hessio_event_source
+from ctapipe.io import CameraGeometry
+
+
+def test_FitGammaHillas():
+
+    filename = get_path("gamma_test.simtel.gz")
+    
+    fit = FitGammaHillas()
+    fit.setup_geometry( *load_hessio(filename),phi=180*u.deg,theta=20*u.deg )
+    
+    tel_geom = {}
+
+    source = hessio_event_source(filename)
+
+    for event in source:
+
+        Hillas_Dict = {}
```

Comment:
use lower-case for variables unless there's a really good reason not to (best to be consistent).  Classes are upper-case + CamelCase, variables are lower-case with underscores.